033-OSSDatabaseMigration/Student/Resources/HelmCharts/ContosoPizza/stop_vmss_node.sh

```diff
@@ -0,0 +1,11 @@
+
+# Stop the VMSS to stop incurring compute charges. There are only two VMSS in the resource group -one each for system and userpool.
+
+
+export vmss_user=$(az vmss list -g MC_OSSDBMigration_ossdbmigration_westus --query '[].name' | grep userpool | tr -d "," | tr -d '"')
+export vmss_system=$(az vmss list -g MC_OSSDBMigration_ossdbmigration_westus --query '[].name' | grep systempool | tr -d "," | tr -d '"')
```

Comment:
These exports are hardcoding the "MC" resource group that the author was using. Either make this more generic or put some comments in here that the value needs to be changed.

Reply:
Fixed